common/src/mailbox_api.rs

```diff
@@ -293,6 +294,28 @@ impl Default for InvokeDpeReq {
     }
 }
 
+// EXTEND_PCR
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct ExtendPcrReq {
+    pub hdr: MailboxReqHeader,
+    pub pcr_idx: u32,
+    pub data: [u8; 48],
+}
+pub type ExtendPcrReqErr = ();
+impl ExtendPcrReq {
+    pub const DATA_MAX_SIZE: usize = 48;
+
+    pub fn new(
+        hdr: MailboxReqHeader,
+        pcr_idx: u32,
+        data: [u8; 48],
+    ) -> Result<Self, ExtendPcrReqErr> {
+        Ok(ExtendPcrReq { hdr, pcr_idx, data })
+    }
+}
```

Comment:
Why have a function for this? It's just filling in a struct with 3 entries with 3 arguments, which is likely inefficient as arguments get copied in functions.

Reply:
This is an artifact from where the default constructor was not suitable, I will make it compact again.